lib/framework.php

```diff
@@ -38,6 +40,11 @@
     require VENDOR_PATH.'paragonie/random_compat/lib/random.php';
 }
 
+/* load env files */
+require_once VENDOR_PATH.'symfony/dotenv/Dotenv.php';
```

Comment:
If you require the vendor/autoload file, you won't need to require individual vendor package classes before using them.

Reply:
Most definitely, the line removed